tests/validation/tests/v3_api/test_bkp_restore_local.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+from .common import *  # NOQA
+
+namespace = {"p_client": None, "ns": None, "cluster": None, "project": None,
+             "nodes": []}
+backup_info = {"backupname": None, "backup_id": None, "workload": None,
+               "backupfilename": None, "etcdbackupdata": None}
+
+
+def test_bkp_restore_local_create():
+    validate_backup_create("local", namespace, backup_info)
```

Comment:
we can remove "local" here as it is not specifically handled in the validate_backup_create method